client/keys/parse.go

```diff
@@ -12,9 +12,10 @@ import (
 	"github.com/spf13/viper"
 	yaml "gopkg.in/yaml.v2"
 
-	"github.com/cosmos/cosmos-sdk/types/bech32"
 	"github.com/tendermint/tendermint/libs/cli"
 
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+
```

Comment:
```suggestion
```

Reply:
oops, this file shouldn't even be in the diff!